Essentials/src/com/earth2me/essentials/commands/Commandafk.java

```diff
@@ -51,7 +51,12 @@ public void run(Server server, CommandSource sender, String commandLabel, String
     private void toggleAfk(User sender, User user, String message) throws Exception {
         if (message != null && sender != null) {
             if (sender.isMuted()) {
-                throw new Exception(tl("voiceSilenced"));
+                if (sender.getMuteReason ().equals ("")) {
```

Comment:
I can't help but feel that there is a better way to format these than adding if/else statements everywhere and concatenating the string inside those blocks. Personally, I would:
* Add a `hasMuteReason` method to `UserData` which does `getMuteReason().equals("")`;
* Use a ternary operator when concatenating the strings instead of if/else blocks:
```java
throw new Exception(tl("voiceSilenced") + (sender.hasMuteReason() ?
                tl("muteReason", sender.getMuteReason())));
```